app/jobs/scheduled/disable_bootstrap_mode.rb

```diff
@@ -0,0 +1,22 @@
+module Jobs
+  class DisableBootstrapMode < Jobs::Scheduled
+    every 1.day
+
+    def execute(args)
+      return unless SiteSetting.bootstrap_mode_enabled
+      total_users = User.where.not(id: Discourse::SYSTEM_USER_ID).count
+
+      if SiteSetting.bootstrap_mode_min_users == 0 || total_users > SiteSetting.bootstrap_mode_min_users
+        update_site_setting('default_trust_level', TrustLevel[0])
+        update_site_setting('default_email_digest_frequency', 10080)
+        update_site_setting('bootstrap_mode_enabled', false)
+      end
+    end
+
+    def update_site_setting(id, value)
```

Comment:
This `update_site_setting` code is the same in two places. I think a class method like `SiteSetting.change(name, val)` might be better and you wouldn't be repeating yourself.